### PR TITLE
Fix TS errors and style problems with the new radios

### PR DESCRIPTION
**Pull request by dabreegster**
It looks like there were a few issues with #56. See comments below

### CODE REVIEW

File: package-lock.json
Comment (dabreegster) [inline package-lock.json:1]: I updated the Radio component in govuk-svelte to use bold labels for consistency with other form elements, and to support the `hint` property. `npm run check` revealed the second problem

File: src/routes/cross_section/data.ts [hunk 1]
```diff
@@ -144,7 +144,7 @@ function emptyState(): State {
 }
 
 export let speedLimits = ["<30", "30", "40", "50", "60", "70"] as const;
-export type SpeedLimit = (typeof speedLimits)[number];
+export type SpeedLimit = (typeof speedLimits)[number] | "";
```
Comment (dabreegster) [inline src/routes/cross_section/data.ts:147]: The empty case was removed above so it didn't show up as a radio button, which is fine. But the initial value is still blank, so TS check broke in a few places

File: src/style/main.sass [hunk 2]
```diff
@@ -28,9 +28,3 @@
   td
     @extend .govuk-table__cell
   // TODO .govuk-table__row for tr isn't found
-
-  .funding-programme
```
Comment (dabreegster) [inline src/style/main.sass:32]: The SASS file is for codebase-wide things, not overrides for particular places

File: src/routes/route_check/summary/+page.svelte [hunk 3]
```diff
@@ -158,12 +155,3 @@
     </ol>
   </ContextualMapEntry>
 </div>
-
-<style>
```
Comment (dabreegster) [inline src/routes/route_check/summary/+page.svelte:162]: This is unused, as `npm run check` reveals

File: src/routes/route_check/summary/+page.svelte [hunk 4]
```diff
@@ -66,16 +66,13 @@
     choices={pairs(regions)}
     bind:value={$state.summary.region}
   />
-  <div class="funding-programme">
```
Comment (dabreegster) [inline src/routes/route_check/summary/+page.svelte:69]: For consistency, removing `inlineSmall` from both for now -- one column with all the choices. It looks a bit awkward. The UX audit has a comment about trying radio buttons in two columns. I'll chat with them tomorrow and figure out what we want to do. But for now, fixing check errors and making things consistent

Comment (Pete-Y-CS) [reply src/routes/route_check/summary/+page.svelte:69]: This was intentional to see which way round users prefer but I'm happy to just choose one and see what feedback comes from that!

File: src/routes/route_check/summary/+page.svelte (hunk 3 above)
Comment (Pete-Y-CS) [reply src/routes/route_check/summary/+page.svelte:162]: I need to get better at remembering to to this. 😩

**Review by Pete-Y-CS — APPROVED** [review]

File: src/routes/route_check/summary/+page.svelte (hunk 3 above)
Comment (dabreegster) [reply src/routes/route_check/summary/+page.svelte:162]: Would it help to set up automatic GH checks? I can do that. I would do the same for running the tests, except they depend on a non-public data input file

Comment (Pete-Y-CS) [reply src/routes/route_check/summary/+page.svelte:162]: Ah okay hmm

**Pull request merged**